_includes/tell-me-why/manage-notifications.md

```diff
@@ -0,0 +1,33 @@
+##Managing Notifications
+Once you've commented on an _issue_ or _pull request_, email notifications will start to pour in. By default notifications will be send by email when there is activity in the thread.  
+You can also choose to receive notifications of a full repository. Whenever there is a new _issue_, _pull request_ or _comment_ posted you will get a notification.
+
+###You can silence or unmute notifications of _individual threads_.
+
+1. Go to the thread.
+2. Click the _(Un)subscribe_ button on the right, under _Notifications_  
+Underneath the button a short description explains the current notification status.
+
+###You can _customize the notifications_ in your settings.
+
+1. Go to your profile icon
+2. Go to _settings_
+3. Select _Notification_ from the menu on the left.  
+Here you can adjust your notification preferences
```

Comment:
We can add a link to the word preferences to this GitHub Help article: https://help.github.com/articles/managing-notification-delivery-methods/